src/extras/TextureTransform.js

```diff
@@ -0,0 +1,115 @@
+import { default as Matrix } from '../core/math/Matrix';
+
+const tempMat = new Matrix();
+
+/**
+ * class controls uv transform and frame clamp for texture
+ */
+export default class TextureTransform {
```

Comment:
Could this be included in TextureUvs? What is the value in this being a separate class?